src/hooks/useMarkdownStyle.ts

```diff
@@ -26,11 +26,13 @@ function useMarkdownStyle(): MarkdownStyle {
             },
             code: {
                 fontFamily: FontUtils.fontFamily.platform.MONOSPACE,
+                fontSize: 13, // TODO: should be 15 if inside h1, see StyleUtils.getCodeFontSize
                 color: theme.text,
                 backgroundColor: 'transparent',
             },
             pre: {
                 fontFamily: FontUtils.fontFamily.platform.MONOSPACE,
+                fontSize: 13,
```

Comment:
I've copied `13` from here:

https://github.com/Expensify/App/blob/7177173f4b686a32bfcb94f07f0541b2f865f31b/src/styles/index.ts#L185-L193

Perhaps we should expose it as `variables.preFontSize`?